trade-finance-manager-ui/server/controllers/case/underwriting/amendments/amendment-underwriter-managers-decision/validateSubmittedValues.test.js

```diff
@@ -0,0 +1,176 @@
+
+    describe('with no validationErrors', () => {
+      it('should return the passed validationErrors and errorsCount', () => {
+        const errors = {
+          testing: true,
+        };
+        const count = 1;
+        const fieldLabel = 'the field label';
+        const fieldId = 'fieldId';
+
+        const result = validateCommentField(
+          errors,
+          count,
+          fieldLabel,
+          fieldId,
+          'valid',
+        );
+
+        const expected = {
+          errorsCount: 1,
+          validationErrors: errors,
+        };
+
+        expect(result).toEqual(expected);
+      });
+    });
+  });
+
+  describe('validateSubmittedValues', () => {
+    it('should return validationErrors when there is no `decision` value', () => {
+      const result = validateSubmittedValues({
+        decision: '',
+      });
+
+      const expected = generateValidationErrors(
+        'decision',
+        'Select if you approve or decline',
+        1,
+      );
+
+      expect(result).toEqual(expected);
+    });
+
+    describe('when `decision` is `Approve with conditions`', () => {
+      it('should return validationErrors when there is no `approveWithConditionsComments`', () => {
+        const result = validateSubmittedValues({
+          decision: 'Approve with conditions',
+          approveWithConditionsComments: '',
+        });
+
+        const expected = generateValidationErrors(
+          'approveWithConditionsComments',
+          'Enter conditions',
+          1,
+        );
+
+        expect(result).toEqual(expected);
+      });
+    });
+
+    describe('when `decision` is `Decline`', () => {
+      it('should return validationErrors when there is no `declineComments`', () => {
+        const result = validateSubmittedValues({
+          decision: 'Decline',
+          declineComments: '',
+        });
+
+        const expected = generateValidationErrors(
+          'declineComments',
+          'Enter reasons',
+          1,
+        );
+
+        expect(result).toEqual(expected);
+      });
+    });
+
+    it('should return multiple validation errors', () => {
+      const result = validateSubmittedValues({
+        decision: '',
+        internalComments: 'a'.repeat(1001),
+      });
+
+      expect(result.count).toEqual(2);
+
+      const decisionError = generateValidationErrors(
+        'decision',
+        'Select if you approve or decline',
+        1,
+      ).errorList.decision;
+
+      expect(result.errorList.decision).toEqual(decisionError);
+
+      const internalCommentsError = generateValidationErrors(
+        'internalComments',
+        'Comments must be 1000 characters or fewer',
+        2,
+      ).errorList.internalComments;
+
+      expect(result.errorList.internalComments).toEqual(internalCommentsError);
+    });
+
+    it('should return false when there are no validationErrors', () => {
+      const result = validateSubmittedValues({
+        decision: 'Approved without conditions',
```

Comment:
we need these decisions in constants because they are referenced in quite a few places